Columbus/frontend/src/pages/Home/Home.js

```diff
@@ -42,12 +45,41 @@ const useStyles = makeStyles((theme) => ({
                   tags: ['cat','love','terminal'],
                   multimedia: ['https://images.pexels.com/photos/8264394/pexels-photo-8264394.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=650&w=940']   
                   }
+    const post3 = {owner_username : 'tarkankuzu', 
```

Comment:
Bu constantları sayfa içerisinde tutmak yerine, Home folderı içerisine Home.constants.js dosyası oluşturulabilir. Eğer öyle olursa bu sayfadaki code u okumak daha kolay olur, nasıl taşınacağına dair bir soru olursa yardımcı olabilirim

Reply:
I will put dummy data in a separate file. Thanks for the suggestion!